Header header
std_msgs/Float32 x_min
std_msgs/Float32 y_min
std_msgs/Float32 x_max
std_msgs/Float32 y_max
std_msgs/Int32 cls
std_msgs/Float32 conf

